code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -311,7 +310,8 @@
 		/obj/item/borg/cyborghug/medical,
 		/obj/item/stack/medical/gauze/cyborg,
 		/obj/item/organ_storage,
-		/obj/item/borg/lollipop)
+		/obj/item/borg/lollipop,
+		/obj/item/weldingtool/cyborg)
```

Comment:
I don't think all cyborgs should be so easy to repair themselves, they should at least need another cyborg to do it.

Reply:
This was shot down by Archanial in my first PR because "Cyborgs are so weak anyway". 
I will gladly re-implement this though.

Reply:
Re-implemented unable to self repair, but since all cyborgs have a welder now you no longer have to rely specifically on an engiborg in cases of subverted silicons. 

👍